docker-compose.it.yml

```diff
@@ -55,16 +58,28 @@ services:
   transformation:
     environment:
       AUTH_ENABLED: 'false'
-      MOCK_RECEIVER_HOST: transformation-it
-      MOCK_RECEIVER_PORT: 8081
 
   transformation-it:
     image: open-data-service/ods-main/transformation-it
     build:
       context: ./transformation/integration-test/
     environment:
       TRANSFORMATION_API: http://transformation:8080
-      MOCK_RECEIVER_HOST: transformation-it
+
+   # ----------------- NOTIFICATION SERVICE (PORTS #TODO) --------------------
```

Comment:
Are these port informations still used somewhere, @georg-schwarz ? As far as I know they are outdated and can be removed anyway (in another issue).

Reply:
Do you mean the TRANSFORMATION_API? That's still used to separate local development env from Docker env.

Reply:
No, I mean the port ranges that are specified for each microservice. I was thinking we don't need them anymore since docker-internal addresses are handled by traefik, but I did not know if you were using them locally somewhere ..